api.go

```diff
@@ -30,6 +30,42 @@ const (
 	)
 )
 
+func (h *Headscale) HealthHandler(
+	writer http.ResponseWriter,
+	req *http.Request,
+) {
+	respond := func(err error) {
+		writer.Header().Set("Content-Type", "application/json; charset=utf-8")
+
+		res := struct {
+			Healthy string `json:"healthy"`
```

Comment:
I am happy with this, just wondering, is this a "defined" standard format for health?

I want:
1. Something is easily consumed by standard health checks, like Docker's, SystemD and Kubernetes
2. The closest to a standard as possible